monkey/monkey_island/cc/ui/src/components/ui-components/AdvancedMultiSelect.js

```diff
@@ -53,8 +53,8 @@ class AdvancedMultiSelect extends React.Component {
     // Apparently, you can use additive operators with boolean types. Ultimately,
     // the ToNumber() abstraction operation is called to convert the booleans to
     // numbers: https://tc39.es/ecma262/#sec-tonumeric
-    if (this.isSafe(b.value) - this.isSafe(a.value) !== 0) {
-      return this.isSafe(b.value) - this.isSafe(a.value);
+    if (this.isSafe(a.value) - this.isSafe(b.value) !== 0) {
+      return this.isSafe(a.value) - this.isSafe(b.value);
```

Comment:
Wait, doesn't this change the order of safe coming after unsafe?

Reply:
Yes. I'm hoping if they float to the top, they won't be checked and hidden down at the bottom. I can revert it if you don't like it.

Reply:
In that case it's probably worth leaving a comment about why we sort this way.

Reply:
Done.